target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-mr8300.dts

```diff
@@ -0,0 +1,400 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/*
+ * Device Tree Source for Linksys MR8300 (Dallas)
+ *
+ * Copyright (C) 2019 Jeff Kletsky
+ * Updated 2020 Hans Geiblinger (support for mr8300)
+ *
+ */
```

Comment:
Since this is almost identical to ea8300.dts, please create a shared DTSI instead of adding all definitions here a second time.

Reply:
@adschm I have done what was asked and used a previous change you submitted **_(ipq806x: create shared DTSI for EA7500 v1 and EA8500)_** my baseline to follow.

I am new to git, so I apologize that in order for me to submit this change, the commit history now shows some unintended items with me merging master into my branch during my testing, however the files changes history is correct.